test/2.0/test-middleware-swagger-validator.js

```diff
@@ -443,9 +443,11 @@ describe('Swagger Validator Middleware v2.0', function () {
       cPetStoreJson.paths['/pets/{id}'].get.operationId = 'getPetById';
 
       helpers.createServer([cPetStoreJson], {
-        conrollers: {
```

Comment:
This worked before, because helper.js defines a default handler which does exactly the same (```res.end('OK');```). I fixed it anyway, since it trapped me when re-using the pattern in the new test.